trino-rest-github/src/main/java/pl/net/was/rest/github/GithubRest.java

```diff
@@ -354,6 +355,16 @@
                     new ColumnMetadata("committer_id", BIGINT),
                     new ColumnMetadata("committer_login", VARCHAR),
                     new ColumnMetadata("parent_shas", new ArrayType(VARCHAR))))
+            .put(GithubTable.PULL_STATS, ImmutableList.of(
+                    new ColumnMetadata("owner", VARCHAR),
+                    new ColumnMetadata("repo", VARCHAR),
+                    new ColumnMetadata("pull_number", BIGINT),
```

Comment:
Can be just `number` to make joins slightly easier. We have `pull_number` in other tables to match field names in the API, but this should match the `pulls` table.

Reply:
I think `number` would be too ambigous, it could mean anything. Let's keep the name consistent across tables.